components/LottieAnimations.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { View, StyleSheet } from 'react-native'
+import { DangerZone } from 'expo'
+const { Lottie } = DangerZone
+const SOURCE = require('../assets/animations/load.json')
+
+const LottieAnimations = ({ width, height, source }) => {
+  let animref = useRef(null)
+  useEffect(() => {
+    if (animref && animref.current) animref.current.play()
+  })
```

Comment:
```suggestion
  }, [])
```

> https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect

Le comportement par default et de trigger l'action au moment ou le composant se monte, mais aussi à chaque update du composant. Pour éviter cela on peut utiliser le deuxième argument qui permet de trigger l'action en fonction de condition (vide donc jamais trigger).

Quoi qu'il arrive l'action est trigger au mount du composant ;-)

Tout est bien expliqué dans la doc 👆

Reply:
merci ;)